package.json

```diff
@@ -11,21 +11,25 @@
   ],
   "scripts": {
     "analyze": "size-limit --why",
-    "build": "dts build",
-    "lint": "dts lint --config \"./.prettierrc.yml\"",
-    "prepare": "dts build",
+    "build": "yarn build:cjs && yarn build:esm",
+    "build:cjs": "tsc --project ./tsconfig.json",
+    "build:esm": "tsc --project ./tsconfig.esm.json",
+    "lint": "eslint --config \"./.eslintrc.json\" \"**/*.{json,ts,yml}\"",
+    "prettier": "prettier --config \"./.prettierrc.json\" --write \"**/*.{json,md,js,ts,yml}\"",
```

Comment:
Nit, the \" characters aren't necessary here because you can just reference the file path directly (not in quotes)

Reply:
👍🏼